src/pages/ClaimsPage/queries.ts

```diff
@@ -4,6 +4,31 @@ import { Contracts } from 'lib/vaults';
 
 import type { Awaited } from 'types/shim';
 
+export const getLockedTokenGifts = async (profileId: number) => {
+  const { locked_token_distribution_gifts } = await client.query({
+    locked_token_distribution_gifts: [
+      {
+        where: {
+          profile_id: { _eq: profileId },
```

Comment:
you can only see rows that you own, due to the row select permissions in Hasura, so this profile_id check isn't needed